cloud-on-k8s-nightly is failing

I broke the job with #1908 (by updating the way we manage the addition of the `push.` prefix to release the operator Docker image),
attempted to fix it with #2135 (by propagating the `REGISTRY` env var),
but it is still broken.

```md
10:43:01  The push refers to repository [docker.elastic.co/eck-snapshots/eck-operator]
10:43:01  15b2eb68b7fb: Preparing 
10:43:01  denied: requested access to the resource is denied
10:43:01  Makefile:332: recipe for target 'docker-push' failed
10:43:01  make[1]: *** [docker-push] Error 1
```

--- a/build/ci/e2e/GKE_k8s_versions.jenkinsfile
+++ b/build/ci/e2e/GKE_k8s_versions.jenkinsfile
@@ -99,10 +99,10 @@ def runWith(lib, failedTests, clusterVersion, clusterName) {
     sh """#!/bin/bash
 
         cat >.env <<EOF
-GCLOUD_PROJECT = "$GCLOUD_PROJECT"
-OPERATOR_IMAGE = "$IMAGE"
+GCLOUD_PROJECT = $GCLOUD_PROJECT
+OPERATOR_IMAGE = $IMAGE
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 SKIP_DOCKER_COMMAND = true
 E2E_JSON = true
 TEST_LICENSE = /go/src/github.com/elastic/cloud-on-k8s/build/ci/test-license.json

--- a/build/ci/e2e/Jenkinsfile
+++ b/build/ci/e2e/Jenkinsfile
@@ -50,9 +50,9 @@ pipeline {
             steps {
                 sh """
                     cat >.env <<EOF
-GCLOUD_PROJECT = "$GCLOUD_PROJECT"
+GCLOUD_PROJECT = $GCLOUD_PROJECT
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 SKIP_DOCKER_COMMAND = false
 TEST_LICENSE = /go/src/github.com/elastic/cloud-on-k8s/build/ci/test-license.json
 GO_TAGS = release

--- a/build/ci/e2e/custom_operator_image.jenkinsfile
+++ b/build/ci/e2e/custom_operator_image.jenkinsfile
@@ -35,10 +35,10 @@ pipeline {
             steps {
                 sh """
                     cat >.env <<EOF
-GCLOUD_PROJECT = "$GCLOUD_PROJECT"
-OPERATOR_IMAGE = "$IMAGE"
+GCLOUD_PROJECT = $GCLOUD_PROJECT
+OPERATOR_IMAGE = $IMAGE
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 IMG_SUFFIX = -ci
 SKIP_DOCKER_COMMAND = true
 E2E_JSON = true

--- a/build/ci/e2e/stack_versions.jenkinsfile
+++ b/build/ci/e2e/stack_versions.jenkinsfile
@@ -141,12 +141,12 @@ pipeline {
 def runWith(lib, failedTests, clusterName, stackVersion) {
     sh """
         cat >.env <<EOF
-OPERATOR_IMAGE = "${IMAGE}"
-GCLOUD_PROJECT = "$GCLOUD_PROJECT"
-STACK_VERSION = "${stackVersion}"
+OPERATOR_IMAGE = ${IMAGE}
+GCLOUD_PROJECT = $GCLOUD_PROJECT
+STACK_VERSION = ${stackVersion}
 SKIP_DOCKER_COMMAND = true
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 E2E_JSON = true
 TEST_LICENSE = /go/src/github.com/elastic/cloud-on-k8s/build/ci/test-license.json
 GO_TAGS = release

--- a/build/ci/e2e/vanilla_k8s.jenkinsfile
+++ b/build/ci/e2e/vanilla_k8s.jenkinsfile
@@ -82,11 +82,11 @@ pipeline {
 def runTests(kindImage) {
     sh """
         cat >.env <<EOF
-OPERATOR_IMAGE = "${IMAGE}"
-GCLOUD_PROJECT = "$GCLOUD_PROJECT"
+OPERATOR_IMAGE = ${IMAGE}
+GCLOUD_PROJECT = $GCLOUD_PROJECT
 SKIP_DOCKER_COMMAND = true
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 E2E_JSON = true
 KIND_NODE_IMAGE = ${kindImage}
 TEST_LICENSE = /go/src/github.com/elastic/cloud-on-k8s/build/ci/test-license.json

--- a/build/ci/pr/Jenkinsfile
+++ b/build/ci/pr/Jenkinsfile
@@ -126,7 +126,7 @@ void createConfig() {
     sh """
         cat >.env <<EOF
 REGISTRY = eu.gcr.io
-REPOSITORY = "$GCLOUD_PROJECT"
+REPOSITORY = $GCLOUD_PROJECT
 TESTS_MATCH = TestSmoke
 SKIP_DOCKER_COMMAND = false
 IMG_SUFFIX = -ci

--- a/build/ci/release/Jenkinsfile
+++ b/build/ci/release/Jenkinsfile
@@ -31,10 +31,10 @@ pipeline {
                         sh """
                             cat >.env <<EOF
 VERSION = ${TAG_NAME}
-REGISTRY = "${REGISTRY}"
-REPOSITORY = "${REPOSITORY}"
-IMG_NAME = "${IMG_NAME}"
-OPERATOR_IMAGE = "${REGISTRY}/${REPOSITORY}/${IMG_NAME}:${TAG_NAME}"
+REGISTRY = ${REGISTRY}
+REPOSITORY = ${REPOSITORY}
+IMG_NAME = ${IMG_NAME}
+OPERATOR_IMAGE = ${REGISTRY}/${REPOSITORY}/${IMG_NAME}:${TAG_NAME}
 SNAPSHOT = false
 GO_TAGS = release
 export LICENSE_PUBKEY = /go/src/github.com/elastic/cloud-on-k8s/build/ci/license.key

